parlai/core/torch_ranker_agent.py

```diff
@@ -119,8 +112,16 @@ def __init__(self, opt, shared=None):
                 broadcast_buffers=False,
             )
 
-    def score_candidates(self, batch, cand_vecs):
-        """Given a batch and candidate set, return scores (for ranking)"""
+    def score_candidates(self, batch, cand_vecs, cand_encs=None):
+        """Given a batch and candidate set, return scores (for ranking).
+
+        :param batch:       a Batch object (defined in torch_agent.py)
```

Comment:
Mind if we use the new docstring format?

https://github.com/facebookresearch/ParlAI/blob/baa839db2120c837562620386cbe1167bd0e9109/parlai/core/torch_agent.py#L650-L659